src/go/rpk/pkg/cli/redpanda/mode.go

```diff
@@ -63,3 +63,27 @@ func executeMode(fs afero.Fs, p *config.Params, mode string) error {
 	}
 	return cfg.SetMode(fs, mode)
 }
+
+const modeHelpText = `Enable a default configuration mode
+
+This command allows you to set one of the following modes: Development,
+Production, or Recovery.
+
+PRODUCTION
+
+  - Enforces optimal runtime checks.
+  - Enables all rpk tuners, for you to run 'rpk redpanda tune'.
+
+DEVELOPMENT
+
+  - Disables optimal checks and bypasses fsync, which results in unrealistically
+    fast clusters and may result in data loss.
+  - Sets all rpk tuners to false. The tuners are intended to run only for
```

Comment:
```suggestion
  - Disables all rpk tuners. The tuners are intended to run only for
```
For consistency with the way we describe production mode i.e. "Enables all rpk tuners"